packages/create-cloudflare/templates/openapi/ts/tsconfig.json

```diff
@@ -1,32 +1,30 @@
 {
 	"compilerOptions": {
-		"allowJs": true,
-		"allowSyntheticDefaultImports": true,
-		"baseUrl": "src",
-		"declaration": true,
-		"sourceMap": true,
+		/* Base Options: */
 		"esModuleInterop": true,
-		"inlineSourceMap": false,
-		"lib": ["esnext"],
-		"listEmittedFiles": false,
-		"listFiles": false,
-		"moduleResolution": "node",
-		"noFallthroughCasesInSwitch": true,
-		"pretty": true,
-		"resolveJsonModule": true,
-		"rootDir": ".",
 		"skipLibCheck": true,
-		"strict": false,
-		"traceResolution": false,
-		"outDir": "",
-		"target": "esnext",
-		"module": "esnext",
+		"target": "es2022",
+		"verbatimModuleSyntax": false,
+		"allowJs": true,
+		"resolveJsonModule": true,
+		"moduleDetection": "force",
+		/* Strictness */
+		"noImplicitAny": false,
+		"noImplicitThis": true,
+		"strictNullChecks": false,
+		"strict": true,
+		"noUncheckedIndexedAccess": true,
+		/* If NOT transpiling with TypeScript: */
+		"moduleResolution": "Bundler",
+		"module": "es2022",
+		"noEmit": true,
+		/* If your code runs in the DOM: */
```

Comment:
Our code doesn't run in the DOM, so I'm not sure what this comment is referring to?

Reply:
this was taken from a tsconfig template, i've removed the comment